sync/class.jetpack-sync-actions.php

```diff
@@ -440,26 +440,34 @@ static function cleanup_on_upgrade( $new_version = null, $old_version = null ) {
 	}
 
 	static function get_sync_status() {
+		require_once JETPACK__PLUGIN_DIR . 'sync/class.jetpack-sync-wp-replicastore.php';
 		self::initialize_sender();
 
 		$sync_module     = Jetpack_Sync_Modules::get_module( 'full-sync' );
 		$queue           = self::$sender->get_sync_queue();
 		$full_queue      = self::$sender->get_full_sync_queue();
 		$cron_timestamps = array_keys( _get_cron_array() );
 		$next_cron       = $cron_timestamps[0] - time();
+		$store           = new Jetpack_Sync_WP_Replicastore();
+		$checksums       = $store->checksum_all();
 
 		$full_sync_status = ( $sync_module ) ? $sync_module->get_status() : array();
+
 		return array_merge(
 			$full_sync_status,
 			array(
-				'cron_size'            => count( $cron_timestamps ),
-				'next_cron'            => $next_cron,
-				'queue_size'           => $queue->size(),
-				'queue_lag'            => $queue->lag(),
-				'queue_next_sync'      => ( self::$sender->get_next_sync_time( 'sync' ) - microtime( true ) ),
-				'full_queue_size'      => $full_queue->size(),
-				'full_queue_lag'       => $full_queue->lag(),
-				'full_queue_next_sync' => ( self::$sender->get_next_sync_time( 'full_sync' ) - microtime( true ) ),
+				'posts_checksum'        => (string) $checksums['posts'],
```

Comment:
Should we add those to the `test_full_sync_status_after_start` test or any other test?

Reply:
Good call. I'll have a look at it tomorrow morning. Still struggling with that `"0"` value for the `comment_meta_checksum` and I have no idea about what could be causing it.

Reply:
Ah, that one. I guess you don't have any comment meta within the comment meta whitelist. How to fix that easily:

* Enable the comments Jetpack module in Discussion settings for Jetpack.
* Go to a post in the frontend while logged into WP.com.
* Leave a comment.
* Re-run the sync status request.